src/main/java/com/gorlah/kappabot/message/MarkdownVariables.java

```diff
@@ -0,0 +1,69 @@
+package com.gorlah.kappabot.message;
+
+public class MarkdownVariables {
+
+    protected static final String ITALICS_BEGIN_VAR                 = "markdown.italics.begin";
```

Comment:
These should probably be package-private instead of `protected`. I think the main difference between package-private and `protected` is that `protected` allows subclass access in addition to everything that package-private allows, which is moot in this case because this class is not extendable due to its private constructor.